src/ExtranetBundle/LoginByTransferToken/TransferTokenService.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace ChameleonSystem\ExtranetBundle\LoginByTransferToken;
+
+use ChameleonSystem\CoreBundle\Interfaces\TimeProviderInterface;
+use Psr\Log\LoggerInterface;
+
+class TransferTokenService implements TransferTokenServiceInterface
```

Comment:
License header is missing.

Reply:
:heavy_check_mark: Added license headers in all files